### ISSUE TITLE
[Bug]: Competitions History refund status

### ISSUE THREAD

**Issue by jonsnowpt** [Betarena/scores#1704]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### 🐞 Describe the bug

"Refund" result is missing.

![Screenshot 2023-10-01 at 15 42 54](https://github.com/Betarena/scores/assets/37311649/01784be3-e030-4619-9981-81629ef7cc27)

### 🤔 Expected Behavior

The refund icon should be gray, similar to this.

<img width="153" alt="Screenshot 2023-10-01 at 15 43 32" src="https://github.com/Betarena/scores/assets/37311649/8709937b-299f-44f7-8975-be1d5587d2be">

### What environment is the bug/issue detected in.

- [ ] Production (scores.betarena.com)
- [X] Development

### What browsers are you seeing the problem on?

- [X] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge
- [ ] Brave
- [ ] Opera

### What device/platform can the problem be seen?

- [ ] 📱 Mobile
- [ ] 💻 Tablet
- [X] 🖥️ Laptop/Desktop

**Comment by migbash** [comment]
"Refund" result is missing.

@jonsnowpt for `(column) result` ? Or `(column) status` ?

Also, the term `refund` needs to be added to the translations, in this case, as it's missing.

**Comment by migbash** [comment]
Added: 

@jonsnowpt Can you add also the missing past tense translations for `won` and `lost` and update them to be in a single `object`, like so:

```json
result:
{
  "refund": "Refunded",
  "won": "Won",
  "lost": "Lost"
}
```

to `betarena_general_data/translations/competitions`.

**Comment by jonsnowpt** [comment]
@migbash 

Done

**Comment by migbash** [comment]
> **Note**
> Created offspring task ➤ #<-github-issue-number->.

**Comment by migbash** [comment]
> **Note**
> Already addressed as part of issue ➤ [#255](https://github.com/Betarena/scores-lib/issues/255).
> Implemented, awaiting `🚀 deployment`.
